packages/fluentui/react-northstar/src/components/Button/ButtonGroup.tsx

```diff
@@ -68,7 +69,7 @@ class ButtonGroup extends UIComponent<WithAsProp<ButtonGroupProps>, any> {
     return (
       <ElementType {...unhandledProps} className={classes.root}>
         {_.map(buttons, (button, idx) =>
-          Button.create(button, {
+          createShorthand<WithAsProp<ButtonProps>>(Button, button, {
```

Comment:
I expect that this will be inferred from the first param

Reply:
Make sense, I changed it